Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -1809,6 +1810,34 @@ def com_google_fonts_check_metadata_subsets_order(family_metadata):
     yield PASS, "METADATA.pb subsets are sorted in alphabetical order."
 
 
+@check(
+  id = 'com.google.fonts/check/metadata/includes_production_subsets',
+  conditions = ['family_metadata', 'production_metadata', 'listed_on_gfonts_api'],
+  rationale = """
+     Check METADATA.pb file includes the same subsets as the family in
+     production.
+  """,
```

Comment:
please do not break lines on rationale strings, as fontbakery itself with do word-wrapping when building its text-terminal UI

Reply:
Ok cool.

Reply:
also, you have 5 spaces here, ideally we should use 4 spaces.